packages/protocol/test/common/integration.ts

```diff
@@ -564,3 +572,187 @@ Array.from([
     })
   })
 )
+
+// TODO: DO NOT FORGET TO REMOVE THE ONLYS HERE
+contract('Integration: Adding StableToken', (accounts: string[]) => {
+  const Exchange: ExchangeContract = artifacts.require('Exchange')
+  const StableToken: StableTokenContract = artifacts.require('StableToken')
+  let exchangeAbc: ExchangeInstance
+  let freezer: FreezerInstance
+  let goldToken: GoldTokenInstance
+  let stableTokenAbc: StableTokenInstance
+  const sellAmount = web3.utils.toWei('0.1', 'ether')
+  const minBuyAmount = 1
+
+  // 0. Make ourselves the owner of the various contracts we will need to interact with, as
+  // passing a governance proposal for each one will be a pain in the butt.
+  before(async () => {
+    goldToken = await getDeployedProxiedContract('GoldToken', artifacts)
+    freezer = await getDeployedProxiedContract('Freezer', artifacts)
+    const governance: GovernanceInstance = await getDeployedProxiedContract('Governance', artifacts)
+    const lockedGold: LockedGoldInstance = await getDeployedProxiedContract('LockedGold', artifacts)
+    const multiSig: GovernanceApproverMultiSigInstance = await getDeployedProxiedContract(
+      'GovernanceApproverMultiSig',
+      artifacts
+    )
+    const registry: RegistryInstance = await getDeployedProxiedContract('Registry', artifacts)
+
+    const proposalId = 1
+    const dequeuedIndex = 0
+    // @ts-ignore
+    await lockedGold.lock({ value: '10000000000000000000000000' })
+    const contractsToOwn = [
+      'Freezer',
+      'Registry',
+      'Reserve',
+      'SortedOracles',
+      'FeeCurrencyWhitelist',
+    ]
+    const proposalTransactions = await Promise.all(
+      contractsToOwn.map(async (x: string) => {
+        return {
+          value: 0,
+          destination: (await getDeployedProxiedContract(x, artifacts)).address,
+          data: Buffer.from(
+            stripHexEncoding(
+              // Any contract's `transferOwnership` function will work here as the function signatures are all the same.
+              // @ts-ignore
+              registry.contract.methods.transferOwnership(accounts[0]).encodeABI()
+            ),
+            'hex'
+          ),
+        }
+      })
+    )
+    await governance.propose(
+      proposalTransactions.map((x: any) => x.value),
```

Comment:
Is the any necessary? Shouldn't it be typed from above?

Reply:
I am not sure what you mean, this is basically a copy&paste from here https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/test/common/integration.ts#L153

Reply:
this should be used in favor of this entire before hook https://github.com/celo-org/celo-monorepo/blob/a4fe8d89b05b67dd889a6c79d2dc1056c6f4a037/packages/protocol/lib/test-utils.ts#L435

Reply:
Thanks @yorhodes, will refactor